run/test/specs/group_disappearing_messages_voice.spec.ts

```diff
@@ -9,23 +10,19 @@ import { setDisappearingMessage } from './utils/set_disappearing_messages';
 iosIt('Disappearing voice message to group', disappearingVoiceMessageGroup);
 androidIt('Disappearing voice message to group', disappearingVoiceMessageGroup);
 
-// bothPlatformsIt(
-//   "Send disappearing voice message to group",
-//   disappearingVoiceMessageGroup
-// );
-
 async function disappearingVoiceMessageGroup(platform: SupportedPlatformsType) {
   const { device1, device2, device3 } = await openAppThreeDevices(platform);
   const testGroupName: GroupName = 'Testing voice';
+  const time = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

Comment:
In `run/test/specs/group_disappearing_messages_video.spec.ts` these are oustide the function? I guess you should do the same here.  I don't have a preference aside from the fact it should be consistent.

Reply:
Has this been been actions? @burtonemily